framework/contracts/account/manager/src/contract.rs

```diff
@@ -120,6 +117,11 @@ pub fn instantiate(
             .add_attribute(install_attribute.key, install_attribute.value);
     }
 
+    if msg.account_id.is_remote() {
+        let install_client_msg = install_ibc_client(deps.branch(), proxy_addr)?;
+        response = response.add_message(install_client_msg);
+    }
+
```

Comment:
Can't we avoid extra message send in case !`install_modules.is_empty()`? Also, what would happen if user decides to specify install of the `ibc-client` for remote account. For example user wants to install lower version of the ibc-client for his remote account for whatever reason

Reply:
You are right, let's push that inside the client API and not inside the contract.

Reply:
> You are right, let's push that inside the client API and not inside the contract.

Do we need to check it inside contract, or it can be installed later?

Reply:
no it doesn't have to be inside contract. For now it's inside the client and the account interface.